test/unit-test/main.tf

```diff
@@ -57,3 +57,56 @@ module "s3_with_notification" {
   notification_sns_arn = aws_sns_topic.topic.arn
   tags                 = local.tags
 }
+
+data "aws_caller_identity" "current" {}
```

Comment:
Can this be moved out of variables.tf into either main.tf, or a dedicated data.tf file please?